Cython/Utils.py

```diff
@@ -54,6 +54,20 @@ def wrapper(*args):
     return wrapper
 
 
+def clear_method_caches(instance):
```

Comment:
Is `Cython/Utils.py` being tested somewhere? It's just that I haven't added any tests for `clear_method_caches`

Reply:
Sort-of, although not much. There's `Cython/Tests/TestCythonUtils.py`.

Reply:
@scoder, It may be I should say that I am writing these tests, but you shouldn't have to resolve this conversation

Reply:
[enough](https://github.com/cython/cython/pull/4241#issuecomment-878709162) utils tests for this pr
basically test are [here](https://github.com/cython/cython/pull/4241/commits/35035f363004fee16314e88c20dc68deafc636f2), [here](https://github.com/cython/cython/pull/4241/commits/14079b723b2baaea79cd417ce53eee9c51da072b), [here](https://github.com/cython/cython/pull/4241/commits/b3eecfbbeb632977525453fe5dbb323a05e281e6), [here](https://github.com/cython/cython/pull/4241/commits/6ea3c569dd1482521fff9b58e76ac2e314a588cd), and [here](https://github.com/cython/cython/pull/4241/commits/1edc3543105beaa2e7d3e15b63131ab7d269d81a)